common/src/main/java/org/apache/uniffle/common/config/ConfigUtils.java

```diff
@@ -207,6 +207,8 @@ && isFinal(modifiers) && field.getType().isAssignableFrom(ConfigOption.class)) {
 
   public static final Function<Long, Boolean> NON_NEGATIVE_LONG_VALIDATOR = value -> value >= 0;
 
+  public static final Function<Integer, Boolean> SERVER_PORT_VALIDATOR = value -> ((value == -1 || value == 0)
```

Comment:
This value can't be `-1` in the configuration. `-1` is our default value, but we can't use it as a configuration value.

Reply:
ok.I'll fix it.